api/v1beta1/safemode.go

```diff
@@ -12,6 +12,7 @@ type UnsafemodeSpec struct {
 	DisableCountTooLarge       bool    `json:"disableCountTooLarge,omitempty"`
 	DisableNeitherHostNorPort  bool    `json:"disableNeitherHostNorPort,omitempty"`
 	DisableSpecificContainDisk bool    `json:"disableSpecificContainDisk,omitempty"`
+	DisableDiskFailurePath     bool    `json:"disableDiskFailurePath,omitempty"`
```

Comment:
/cc @Azoam 

It goes against consistency, but would naming this something like `AllowRootDiskFailure`, `AllowDiskFailureAtRoot`, etc., be more clear to the users?

Reply:
I am totally agreed with you!  `AllowRootDiskFailure` sounds good to me. 👍   I was not 100% satisfied by my naming.